src/main/java/uk/gov/hmcts/reform/idam/health/command/ReplicationCommandProbe.java

```diff
@@ -145,11 +149,11 @@ protected ReplicationInfo convert(String value) {
             info.setDsID(StringUtils.trimToNull(parts[i++]));
             info.setRsId(StringUtils.trimToNull(parts[i++]));
             info.setRsPort(StringUtils.trimToNull(parts[i++]));
-            String missingChanges = StringUtils.trimToNull(parts[i++]);
-            if (missingChanges != null) {
-                info.setDelay(Integer.parseInt(missingChanges));
+            String delay = StringUtils.trimToNull(parts[i++]);
+            if (delay != null) {
+                info.setDelay(delay.equals("N/A") ? 0 : Integer.parseInt(delay));
             }
-            info.setSecurityEnabled(StringUtils.trimToNull(parts[i++]));
+            info.setSecurityEnabled(StringUtils.trimToNull(parts[i]));
```

Comment:
I know this logic was already there but if we are going to add further updates we might as well refactor this method just to keep our minds healthy 😆